secator/output_types/vulnerability.py

```
@@ -16,9 +16,9 @@ class Vulnerability(OutputType):
 	ip: str = field(default='', compare=False)
 	confidence: str = 'low'
 	severity: str = 'unknown'
-	cvss_score: float = 0
-	cvss_vec: str = ''
-	epss_score: float = 0
+	cvss_score: field(default=0, compare=False)
+	cvss_vec: field(default='', compare=False)
```

Flake8 (7.3.0)

[error] syntax error in forward annotation ''

(F722)